doc/source/user_guide/numpy_images.rst

```diff
@@ -65,7 +65,7 @@ Masking (indexing with masks of booleans)::
 
 Fancy indexing (indexing with sets of indices)::
 
-    >>> inds_r = np.arange(len(camera))
```

Comment:
not explicit enough, the len() operator on a numpy array is confusing and depends on the storage (fortran or C)

Reply:
I kinda believe you that this might be true in some version of numpy. Could you show me some reproducing code? My version of numpy doesn't seem to depend on fortran vs C. I personally prefer `shape[0]` though.

```python
Python 3.6.6 | packaged by conda-forge | (default, Oct 12 2018, 14:08:43) 
[GCC 4.8.2 20140120 (Red Hat 4.8.2-15)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy as np
>>> np.__version__
'1.15.2'
>>> a = np.zeros((3, 4), order='C')
>>> b = np.zeros((3, 4), order='F')
>>> len(a)
3
>>> len(b)
3
```

Reply:
@jni @hmaarrfk My mistake, they indeed overloaded the `len` operator for arrays in numpy, however I do agree that `shape[0]` is more explicit and could be considered here.